make_flaggable.gemspec

```diff
@@ -14,8 +14,8 @@ Gem::Specification.new do |s|
   s.required_rubygems_version = ">= 1.3.6"
   s.rubyforge_project         = "make_flaggable"
 
-  s.add_dependency "activerecord", ['>= 3.0', '< 4.2']
-  s.add_development_dependency "bundler", ['>= 1.0.0', '<= 1.4']
+  s.add_dependency "activerecord", '< 4.3'
```

Comment:
I think we should maintain the `>= 3.0` because I do believe it relies on syntax from AR 3. And we should issue a PR against the original with all these changes. I'd be comfortable we just said `>= 3.0` without the any upper limit.